components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java

```diff
@@ -226,4 +227,31 @@ public static void initiatePortals(String tenantDomain, int tenantId)
             }
         }
     }
+
+    /**
+     * Get OAuth InboundAuthenticationRequestConfig of the application.
+     *
+     * @param application application.
+     * @return OAuth InboundAuthenticationRequestConfig if exists.
+     */
+    public static InboundAuthenticationRequestConfig getOAuthInboundAuthenticationRequestConfig(
+            ServiceProvider application) {
+
+        if (application == null || application.getInboundAuthenticationConfig() == null
+                || application.getInboundAuthenticationConfig().getInboundAuthenticationRequestConfigs() == null
+                || application.getInboundAuthenticationConfig().getInboundAuthenticationRequestConfigs().length == 0) {
+
```

Comment:
do we need these new lines before the return?

Reply:
much readable that way.